splink/linker.py

```diff
@@ -1753,10 +1753,11 @@ def find_matches_to_new_records(
 
         if not isinstance(records_or_tablename, str):
             uid = ascii_uid(8)
+            new_records_tablename = f"__splink__df_new_records_{uid}"
             self.register_table(
-                records_or_tablename, f"__splink__df_new_records_{uid}", overwrite=True
+                records_or_tablename, new_records_tablename, overwrite=True
```

Comment:
I take it this is just general cleaning?

Reply:
Yeah - think I had initially made some changes here such that this seemed like a natural change, but then I ended up amending `ascii_uid` instead. Thought this seemed reasonable enough to leave